_pages/production.md

```diff
@@ -157,8 +157,9 @@ If you wish to deploy an application to our staging environment, then create a "
 
 Changes to production applications
 ----------------------------------
+**Changes to the production app are not automatic and may take up to two weeks**.
```

Comment:
[suggestion] Include the period with the rest of the bolded text.

Reply:
Thank you!